utilities_common/cli.py

```diff
@@ -1,9 +1,23 @@
+import os
+import sys
+import netaddr
+import subprocess
+
 import click
 
 from utilities_common.db import Db
 
+from sonic_py_common import logger
+from swsssdk import ConfigDBConnector
+
+SYSLOG_IDENTIFIER_FOR_CONFIG = "config"
+VLAN_SUB_INTERFACE_SEPARATOR = '.'
+
 pass_db = click.make_pass_decorator(Db, ensure=True)
 
+# Global logger instance
+cfglog = logger.Logger(SYSLOG_IDENTIFIER_FOR_CONFIG)
```

Comment:
This is specific to `config` not sure if it belongs here, as utilities_common will be shared between all the CLI utilities. I think this would be better if placed in a new file *inside* the config/ directory (e.g., config/common.py or config/utils.py) which can be shared between all the .py files which are part of `config`, but *not* shared with other utilities.